File: submissions/0344-reverse-string/solution.ts
/**
 Do not return anything, modify s in-place instead.
 */
function reverseString(s: string[]): void {
    let left:number = 0;
 for(let right = s.length - 1; right>=left; right--){
     let temp:string = s[left]
     s[left] = s[right]
     s[right] = temp
     left++
 }   
};

File: submissions/0344-reverse-string/solution.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int left = 0;
        int right = s.size() - 1;
        
        while(left < right){
            s.insert(s.begin() + left, s[right]);
            s.erase(s.end());
            
            left++;
        }
    }
};
